src/kixi/data_acquisition/db/inmemory.clj

```diff
@@ -0,0 +1,40 @@
+(ns kixi.data-acquisition.db.inmemory
```

Comment:
I more worry about how the structure of something in c* or es can affect how you would model your data and that we might make something that either didn't fit or didn't take advantage of it. I think we are OK for now, but it is a question I think we should ask ourselves when using mocks like this.